src/features/threads/InteractivePostStats.js

```diff
@@ -15,7 +15,8 @@ class InteractivePostStats extends Feature {
     // Loop post author
     $(".post").each(function() {
       const postbitTable = $(this).find(".post_author");
-      const userId = $(postbitTable).find(".author_information .largetext > a").attr("href").split("&uid=")[1];
+      const userId = $(postbitTable).find(".author_information .largetext > a").attr("href")?.split("&uid=")[1];
+      if (!userId) return;
```

Comment:
Have you tested that returning null doesn't break the loop?
The jquery docs say returning false breaks the loop, so I wanted to be sure any users after the guest still function as expected. (Docs: https://api.jquery.com/jQuery.each/)

Reply:
I have not tested it. I looked it up on StackOverflow https://stackoverflow.com/a/17162375
From what I gather:
`return false` == `break`
 `return` == `continue`

Reply:
I'll test it out tomorrow after work, thanks!